ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4827,6 +4829,274 @@ HWRWISOVPM:
+  - supply_fan_run_mode
+  implements:
+  - MONITORING
+
+SRC:
+  description: "Very basic run scheduling command. "
+  is_abstract: true
+  uses:
+  - schedule_run_command
+  implements:
+  - CONTROL
+
+SAIDC:
+  description: "isolation damper status monitoring and control on the supply side."
+  is_abstract: true
+  uses:
+  - supply_air_isolation_damper_status
+  - supply_air_isolation_damper_control  
+  implements:
+  - MONITORING
+
+SAIDC2X:
+  description: "isolation damper status monitoring and control on the supply side, where there are two separate, equal sets of dampers that operate in conjunction."
+  is_abstract: true
+  uses:
+  - supply_air_isolation_damper_status_1
+  - supply_air_isolation_damper_status_2
+  - supply_air_isolation_damper_control_1
+  - supply_air_isolation_damper_control_2  
+  implements:
+  - MONITORING
+
+SAIDC3X:
+  description: "isolation damper status monitoring and control on the supply side, where there are three separate, equal sets of dampers that operate in conjunction."
+  is_abstract: true
+  uses:
+  - supply_air_isolation_damper_status_1
+  - supply_air_isolation_damper_status_2
+  - supply_air_isolation_damper_status_3
+  - supply_air_isolation_damper_control_1
+  - supply_air_isolation_damper_control_2
+  - supply_air_isolation_damper_control_3  
+  implements:
+  - MONITORING   
+  
+SSPM2X:
+  description: "Static pressure monitoring on the supply side where the average of the two sensors are being taken."
+  is_abstract: true
+  uses:
+  - supply_air_static_pressure_sensor_1
+  - supply_air_static_pressure_sensor_2
+  - supply_air_static_pressure_setpoint_1
+  - supply_air_static_pressure_setpoint_2   
+  implements:
+  - MONITORING
+
+SFRDM:  
+  description: "Supply fan run duration monitoring "
+  is_abstract: true
+  uses:
+  - supply_fan_run_time_accumulator
+  implements:
+  - MONITORING 
+
+CPRDM:  
+  description: "Compressor run duration monitoring "
+  is_abstract: true
+  uses:
+  - compressor_run_time_accumulator
+  implements:
+  - MONITORING 
+
+EVFSM:
+  description: "Evaporative fan speed monitoring ."
+  is_abstract: true
+  uses:
+  - evaporative_fan_speed_sensor
+  implements:
+  - CONTROL 
+
+RDM:
+  description: "Very basic system run duration monitoring. "
+  is_abstract: true
+  uses:
+  - run_time_accumulator
+  implements:
+  - MONITORING
+
+ZTLM:
+  description: "Zone temperature setpoint limit monitoring."
+  is_abstract: true
+  uses:
+  - high_limit_zone_air_temperature_setpoint
+  - low_limit_zone_air_temperature_setpoint
+  - zone_air_temperature_setpoint
+  implements:
+  - MONITORING
+  
+SFMSC:
+  description: "Discharge fan multi-speed control."
+  is_abstract: true
+  uses:
+  - supply_fan_run_command
+  - supply_fan_run_status
+  - supply_fan_speed_mode 
+
+SFRDM:
+  description: "Very basic system run duration monitoring for supply fans. "
+  is_abstract: true
+  uses:
+  - supply_fan_run_time_accumulator
+  implements:
+  - MONITORING  
+  
+#HRZAM:
```

Comment:
We will not merge until the commented out types are removed, so please do not forget to remove these.